### ISSUE TITLE
Windows 32bit: HelpFlagsTest fails due to missing specifications

### ISSUE THREAD

**Issue by adamfarley** [adoptium/aqa-tests#2840]
**Describe the bug**
OpenJDK Test tools/launcher/HelpFlagsTest.java fails on Windows 32bit, JDK11 and JDK16, due to an apparent test config issue.

JDK11: https://ci.adoptopenjdk.net/job/Test_openjdk11_hs_extended.openjdk_x86-32_windows_testList_0/10/console
JDK16: https://ci.adoptopenjdk.net/job/Test_openjdk16_hs_extended.openjdk_x86-32_windows_testList_0/7/consoleFull

[Earlier reference.](https://github.com/adoptium/aqa-tests/issues/2382#issuecomment-799548468)

**To Reproduce**
[Grinder link](https://ci.adoptopenjdk.net/job/Grinder/parambuild/?SDK_RESOURCE=upstream&TARGET=-f+parallelList.mk+testList_0&TEST_FLAG=&UPSTREAM_TEST_JOB_NAME=Test_openjdk11_hs_extended.openjdk_x86-32_windows&DOCKER_REQUIRED=false&ACTIVE_NODE_TIMEOUT=0&VENDOR_TEST_DIRS=&EXTRA_DOCKER_ARGS=&TKG_OWNER_BRANCH=AdoptOpenJDK%3Amaster&TEST_PARALLELIZATION_PARAMS=&OPENJ9_SYSTEMTEST_OWNER_BRANCH=eclipse%3Amaster&PLATFORM=x86-32_windows&GENERATE_JOBS=true&KEEP_REPORTDIR=true&PERSONAL_BUILD=false&ADOPTOPENJDK_REPO=https%3A%2F%2Fgithub.com%2Fadoptium%2Faqa-tests.git&LABEL=&TEST_OPTIONS_PARAMS=&EXTRA_OPTIONS=&CUSTOMIZED_SDK_URL=&BUILD_IDENTIFIER=&NON_AQA_TEST_REPOS_HELP_TEXT=&ADOPTOPENJDK_BRANCH=master&LIGHT_WEIGHT_CHECKOUT=false&NON_AQA_TEST_REPOS=&ARTIFACTORY_SERVER=&TEST_REPO_PARAMS=&TEST_SELECTION_PARAMS=&TEST_PARALLELIZATION_PARAMS_HELP_TEXT=&JDK_SELECTION_PARAMS=&KEEP_WORKSPACE=false&USER_CREDENTIALS_ID=&JDK_VERSION=11&ITERATIONS=1&VENDOR_TEST_REPOS=&JDK_REPO=https%3A%2F%2Fgithub.com%2Fadoptium%2Fjdk11u&PLATFORM_AND_MACHINE_HELP_TEXT=&RELEASE_TAG=&OPENJ9_BRANCH=master&OPENJ9_SHA=&JCK_GIT_REPO=&VENDOR_TEST_BRANCHES=&UPSTREAM_JOB_NAME=build-scripts%2Fjobs%2Fjdk11u%2Fjdk11u-windows-x86-32-hotspot&OPENJ9_REPO=https%3A%2F%2Fgithub.com%2Feclipse-openj9%2Fopenj9.git&CLOUD_PROVIDER=&PLATFORM_AND_MACHINE=&CUSTOM_TARGET=&VENDOR_TEST_SHAS=&JDK_BRANCH=dev&LABEL_ADDITION=&ARTIFACTORY_REPO=&ARTIFACTORY_ROOT_DIR=&POST_RUN_PARAMS_HELP_TEXT=&UPSTREAM_TEST_JOB_NUMBER=35&DOCKERIMAGE_TAG=&JDK_SELECTION_PARAMS_HELP_TEXT=&JDK_IMPL=hotspot&TEST_TIME=&SSH_AGENT_CREDENTIAL=&AUTO_DETECT=true&SLACK_CHANNEL=&TKG_SHA=&TEST_SELECTION_PARAMS_HELP_TEXT=&ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH=adoptium%3Amaster&CUSTOMIZED_SDK_URL_CREDENTIAL_ID=&ARCHIVE_TEST_RESULTS=false&NUM_MACHINES=&OPENJDK_SHA=&TRSS_URL=&BUILD_LIST=openjdk&UPSTREAM_JOB_NUMBER=929&STF_OWNER_BRANCH=adoptium%3Amaster&TEST_REPO_PARAMS_HELP_TEXT=&POST_RUN_PARAMS=&TIME_LIMIT=10&TEST_OPTIONS_PARAMS_HELP_TEXT=&JVM_OPTIONS=&PARALLEL=None)

**Expected behavior**
I expect the test to pass, as it does on Windows 64bit.

**Screenshots**
```
23:47:39  STDERR:
23:47:39  java.lang.AssertionError: HelpFlagsTest failed:
23:47:39  Tool jaccessinspector-32.exe not covered by this test. Add specification to jdkTools array!
23:47:39  Tool jaccesswalker-32.exe not covered by this test. Add specification to jdkTools array!
23:47:39  
23:47:39  	at HelpFlagsTest.main(HelpFlagsTest.java:378)
23:47:39  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
23:47:39  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
23:47:39  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
23:47:39  	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
23:47:39  	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:298)
23:47:39  	at java.base/java.lang.Thread.run(Thread.java:829)
23:47:39  
23:47:39  JavaTest Message: Test threw exception: java.lang.AssertionError
23:47:39  JavaTest Message: shutting down test
```

**Additional context**
Test will be excluded. [Link.](https://github.com/adoptium/aqa-tests/pull/2842)

**Comment by smlambert** [comment]
I am presuming the jdkTools array is in upstream code, in which case this issue should get reported and fixed upstream.

**Comment by adamfarley** [comment]
Issue reported here: https://bugs.openjdk.java.net/browse/JDK-8273235

Seems a simple enough fix now I've read the code. Have added notes on the fix to that bug.

**Comment by smlambert** [comment]
Closed now that #3149 is merged.
